### PR TITLE
Correctly handle each asset type

### PR DESCRIPTION
**Pull request by winniederidder**
This pull request tries to fix the asset resolution problem for the input service worker by choosing the correct assets in any environment and obtaining the correct filename.

### CODE REVIEW

File: app/controllers/activities_controller.rb [hunk 1]
```diff
@@ -204,9 +206,13 @@ def media
   # Asset has been preprocessed and built internally
   # Redirecting to the asset is not possible due to browser security policy
   def input_service_worker
-    assets = Rails.application.assets || Rails.application.assets_manifest.assets
-    send_file(assets['inputServiceWorker.js'].filename,
-              filename: 'inputServiceWorker.js',
+    filename = if Rails.application.assets
+                 Rails.application.assets[INPUT_SERVICE_WORKER].filename
+               else
+                 Rails.application.assets_manifest.assets[INPUT_SERVICE_WORKER]
```
Comment (chvp) [inline app/controllers/activities_controller.rb:212]: This returns something of the form `"inputServiceWorker-abadf00e34692d3841e1f735d8f83f1d9879fd0b95ee577ded766655f9d28661.js"`. Just `send_file`-ing that will not work.

**Review by chvp — CHANGES_REQUESTED** [review]

File: app/controllers/activities_controller.rb (hunk 1 above)
Comment (winniederidder) [reply app/controllers/activities_controller.rb:212]: Would something like Rails.root.join("app", "assets", "inputServiceWorker-abadf00e34692d3841e1f735d8f83f1d9879fd0b95ee577ded766655f9d28661.js") work, or will this file also be in the build subdirectory in production?

Comment (chvp) [reply app/controllers/activities_controller.rb:212]: It will be in `public/assets`, but that doesn't really feel like a clean solution to me.

Comment (jorg-vr) [reply app/controllers/activities_controller.rb:212]: `Rails.application.assets_manifest.directory` contains the directory the file is in.

Comment (jorg-vr) [reply app/controllers/activities_controller.rb:212]: https://stackoverflow.com/a/61552216

**Review by chvp — APPROVED** [review]

**Review by bmesuere — APPROVED** [review]

**Review by jorg-vr — APPROVED** [review]

**Review by niknetniko — APPROVED** [review]

**Pull request merged**